### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by jy-yy**

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -64,3 +64,8 @@ int  main() {
 	std::cout << game[2][0] << "\t" << game[2][1] << "\t" << game[2][2] << std::endl;
 	return 1;
 }
+ # new version
+1.修改了输入无限制的问题
```
Comment (NieZhinie) [inline mcr.cpp:68]: nice

**Review by NieZhinie — COMMENTED** [review]
nice

**Review by SharonHYG — COMMENTED** [review]
All the problems have been solved.

**Comment by llllllllydiaaaaaaa** [comment]
well

File: mcr.cpp (hunk 1 above)
Comment (jy-yy) [reply mcr.cpp:68]: thank you!

**Comment by jy-yy** [comment]
> 

yes, thank you very much to review my code

**Pull request merged**